anesthetic/samples.py

```diff
@@ -194,11 +203,35 @@ def plot_2d(self, axes, *args, **kwargs):
             this is used for creating the plot. Otherwise a new set of axes are
             created using the list or lists of strings.
 
-        types: list(str) or str, optional
-            What type (or types) of plots to produce. If two types are provided
-            then pairs of parameters 'above the diagonal' have the second type.
-            each string must be one of {'kde', 'scatter'}.
-            Default: ['kde', 'scatter']
+        types: dict, optional
+            What type (or types) of plots to produce. Takes the keys 'diagonal'
+            for the 1D plots and 'lower' and 'upper' for the 2D plots.
+            The options for 'diagonal are either:
+
+                - 'kde'
+                - 'hist.
+
+            The options for 'lower' and 'upper' are either:
+
+                - 'kde'
+                - 'scatter'
+
```

Comment:
You don't need the extra blank lines here.

Reply:
Was matching it to the itemisation above, but actually agree, I prefer it without the blank lines, too. Changed accordingly.